C++/3. Operators.md

```diff
@@ -2,42 +2,38 @@
 
 ## Start
 
-There are certain operators which do addition, and things like that in C++. In this file, I will teach you the basic operators:
+C++ provides certain operators which do addition, and things like that in C++. In this file, I will teach you the basic arithmetic operators:
 
 - `+`
 - `-`
 - `*`
 - `/`
-- `%`
 
 ## What they do
 
-The `+` operator adds things together. \
-The `-` operator subtracts two things. \
-The `*` operator multiplies things together. \
-The `/` operator divides two things. \
-The `%` operator find the reminder of a division.
```

Comment:
Hey we need this, I use this a lot. Add it back

Reply:
Ha, okay, I'll re-add the `%` operator later.